tests/gcp/00-application.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: skiperator.kartverket.no/v1alpha1
+kind: Application
+metadata:
+  name: test
```

Comment:
namespace test not specififed

Reply:
This is configured in kuttls config for TestSuite here: https://github.com/kartverket/skiperator/blob/main/tests/config.yaml#L23
Docs: https://kuttl.dev/docs/testing/reference.html#testsuite

Since this is configured elsewhere we've chosen to not set this here